README.md

```diff
@@ -21,11 +21,12 @@ Introducing the flagship product of [Ubiquity DAO](https://ubq.fi/). The Ubiquit
 
 git clone https://github.com/ubiquity/ubiquity-dollar.git
 cd ubiquity-dollar/
-yarn
+yarn # fetch dependencies
 yarn build:all # builds the smart contracts and user interface
-yarn start & # starts the user interface and daemonize'd to continue to run tests in the background
+yarn start # starts the user interface and daemonize'd to continue to run tests in the background
```

Comment:
Currently this command fails on a clean checkout as the "start" script definition is not available in the top level package.json and is defined in packages/dapp only.

Reply:
Additionally, when we remove `&` the dapp starts in the foreground mode, therefore everything written after "starts the user interface" can be removed. We should explicitly add though that this command runs in foreground and another shell / console must be open for subsequent commands to run in parallel to the user interface.